libs/feature/record/src/lib/state/mdview.effects.ts

```diff
@@ -32,6 +31,35 @@ export class MdViewEffects {
       )
     )
   )
+  /*
+  Feature Catalog effects
+  */
+  loadCatalogAttributes = createEffect(() =>
```

Comment:
Here I would add an additional effect that emits a `loadFeatureCatalogAttributes` action when a `loadFullMetadataSuccess` action is received; this way the loading of the feature catalog will be done automatically when a full record is available, and then `repository.getFeatureCatalog(record)` can be called